README.md

```diff
@@ -58,6 +58,13 @@ Run the command ```yarn test:jest``` in the OpenSearch Dashboards project direct
 For the integration tests, you must pass the absolute path of your extracted OpenSearch build artifacts through the ```TEST_OPENSEARCH_FROM``` env variable so that Dashboards can run an OpenSearch cluster. You can run:
 ```TEST_OPENSEARCH_FROM=/path/to/extracted/build/artifact yarn test:jest_integration```
 
+## Building the Docker Image
+
+```
+yarn osd bootstrap
+yarn build --docker
+```
+
```

Comment:
Do you feel this be good place inside the Developer Guide ?

Reply:
Not really, the Developer Guide seems to be more focused on code styling and development suggestions, while this may actually be used (as in my case) to build the project and deploy it somewhere :)

Reply:
I can see an argument for putting it in the README. If we feel like it's not the place for it in the future we could potentially move it if we are unsure about placing it in the README @mihirsoni

Reply:
I don't mind, my only points is we have many distribution. It could grow in future saying how to build  `.deb` or `.rpm`. How about instead of making it docker specific, we make it generalized  with `yarn build --help` and have a line below as just one example and communication for other build types explore `yarn build --help` command. 

Thoughts @kavilla @denysvitali

Reply:
That would be an option, yes. But I would argue that nowadays Docker is the most portable solution and should be prioritized over the alternatives (`rpm`s, `deb`s). Usually, the first thing I look for a project (especially _this size_) is a Docker Image reference / Dockerfile